### PR TITLE
Initialize DRF app

### PR DESCRIPTION
**Pull request by tlindaliu**
@BenjiLee 
 FYI @jimabramson, @dan-f

### CODE REVIEW

File: discussions/settings.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+WSGI_APPLICATION = 'discussions.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/1.8/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```
Comment (BenjiLee) [inline discussions/settings.py]: Note: We will probably end up changing this.

File: discussions/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (BenjiLee) [inline discussions/tests.py]: This file will probably not exist if we choose to make a tests folder which I think we are going to.

**Comment by BenjiLee** [comment]
Only notes: Otherwise :+1:

**Comment by clintonb** [comment]
See edx/ecommerce or edx/edx-analytics-dashboard for examples of setting up projects for multiple environments.

**Comment by jimabramson** [comment]
need to ignore / remove *.pyc

**Comment by jimabramson** [comment]
to @clintonb's advice about settings: we can probably go with base, local, & test settings modules for the moment.  to bootstrap things put everything in base and have test and local `import *`

**Comment by jimabramson** [comment]
note we should also have a Makefile patterned after the one in ecommerce (albeit much simpler); however that can be in a separate PR.

**Comment by jimabramson** [comment]
:+1: (unless you plan to change settings to a directory in this PR?)

File: discussions/settings.py (hunk 1 above)
Comment (dan-f) [inline discussions/settings.py]: We should just delete this, since we're using mongo.

**Comment by tlindaliu** [comment]
added the settings directory

**Comment by jimabramson** [comment]
:+1:  please squash the commits before merging

**Comment by BenjiLee** [comment]
:+1:

**Pull request merged**